external/jsdiff/jsdiff.js

```diff
@@ -1,148 +1,1071 @@
 // jscs:disable maximumLineLength
 /*
- * Javascript Diff Algorithm
- *  By John Resig (http://ejohn.org/)
- *  Modified by Chu Alan "sprite"
+ *	Diff Algorithm by Google's Diff-Match-Patch library
  *
- * Released under the MIT license.
+ * Copyright 2006 Google Inc.
+ * http://code.google.com/p/google-diff-match-patch/
+ *
+ * Released under the Apache License, Version 2.0 (the "License");
  *
  * More Info:
- *  http://ejohn.org/projects/javascript-diff-algorithm/
+ *  https://code.google.com/p/google-diff-match-patch/
```

Comment:
It might be possible to paste the license inline here, I'm not sure about it. Would you confirm this, @scottgonzalez?